app/src/components/Login.js

```diff
@@ -1,11 +1,16 @@
 import React from "react";
 
 import { useAuth0 } from "@auth0/auth0-react";
+import Button from "react-bootstrap/Button";
 
 const LoginButton = () => {
   const { loginWithRedirect } = useAuth0();
 
-  return <button onClick={() => loginWithRedirect()}>Log In</button>;
+  return (
+    <Button variant="light" size="sm" onClick={() => loginWithRedirect()}>
```

Comment:
If your callback method doesn't need access to any local variable, you could just directly pass the function as the callback, like this:
```
    <Button {/* ... */} onClick={loginWithRedirect}>
```